skimage/_shared/utils.py

```diff
@@ -113,6 +114,61 @@ def fixed_func(*args, **kwargs):
         return fixed_func
 
 
+def docstring_add_deprecated(func, kwarg_mapping, deprecated_version=None):
+    """Add deprecated kwarg(s) to the "Other Params" section of a docstring.
+
+    Parmeters
+    ---------
+    func : function
+        The function whos docstring we wish to update.
+    kwarg_mapping : dict
+        A dict containing {old_arg: new_arg} key/value pairs as used by
+        `deprecate_kwarg`.
+    deprecated_version : str, optional
+        A major.minor version string specifying when old_arg was
+        deprecated.
+
+    Returns
+    -------
+    new_doc : str
+        The updated docstring. Returns the original docstring if numpydoc is
+        not available.
+    """
+    try:
+        from numpydoc.docscrape import FunctionDoc, Parameter
+    except ImportError:
+        # Return an unmodified docstring if numpydoc is not available.
+        return func.__doc__
+
+    Doc = FunctionDoc(func)
```

Comment:
```suggestion
    Doc = FunctionDoc(func)
    if Doc is None:
        return None
```

Don't you need this for PYTHON_OPTIMIZED builds

Reply:
Thanks, this suggestion is now covered by the if `func.__doc__ is None` above